src/main/java/Main.java

```diff
@@ -0,0 +1,11 @@
+import controller.RaceController;
+import utils.RandomDigitGenerator;
+import view.RaceInputView;
+import view.RaceOutputView;
+
+public class Main {
+    public static void main(String[] args) {
+        new RaceController(new RaceInputView(), new RaceOutputView(), new RandomDigitGenerator())
```

Comment:
`RaceController` 생성자 부분에서 지금은 파라미터가 3개라 괜찮지만, 나중에 더 늘어날 가능성을 고려해서 가독성을 미리 확보하는 차원에서 아래와 같이 작성하는 건 어떨까요?

```suggestion
        new RaceController(
            new RaceInputView(), 
            new RaceOutputView(), 
            new RandomDigitGenerator()
        )
```

이렇게 하면 나중에 의존성이 추가/변경될 때 동현님뿐만 아니라 다른 팀원들도 어떤 의존성이 필요한지 한눈에 파악할 수 있어 유지보수에 큰 도움이 될 거라고 생각하는데, 동현님의 생각은 어떠신가요?

Reply:
좋은 생각입니다! 수정했습니다!

또한 자동으로 스타일을 맞추도록 위와 같이 코드 스타일 설정을 수정하였습니다!
<img width="343" height="228" alt="image" src="https://github.com/user-attachments/assets/eff0f750-ae02-46d2-9d9e-ada0bdec8e84" />